package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "temporal_tables",
+  "version": "0.4.0",
+  "description": "![](https://github.com/nearform/temporal_tables/workflows/ci/badge.svg)",
```

Comment:
why the badge in the description?

Reply:
for some reason, this was generated automatically when I ran the command `npm init --y`. I guess there is no need for it, I'll remove it.